job-server/src/main/resources/application.conf

```diff
@@ -17,6 +17,12 @@ spark {
       rootdir = /tmp/spark-jobserver/filedao/data
     }
 
+    # To load up job jars on startup, place them here,
+    # with the app name as the key and the path to the jar as the value
+    # job-jar-uris {
+    #   test = ../job-server-tests/target/scala-2.10/job-server-tests_2.10-0.5.3-SNAPSHOT.jar
```

Comment:
Can this be any URI? For example we store job jar in an HTTP server. Will this patch help us to put the url here and upload the jar?